core/templates/domain/topic/TopicObjectFactory.ts

```diff
@@ -113,8 +123,8 @@ export class Topic {
     this._pageTitleFragmentForWeb = pageTitleFragmentForWeb;
   }
 
-  // ---- Instance methods ----
-  getId(): string {
+  // Returns 'null' when the topic is not yet saved on the backend.
```

Comment:
It is null when the topic is not yet fetched from the backend, not saved.

Reply:
Done